include/swift/Parse/Confusables.def

```diff
@@ -0,0 +1,119 @@
+//===--- Confusables.def - Swift Tokenizer Metaprogramming ------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines macros used for diagnosing confusable Unicode characters.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CONFUSABLE
+#define CONFUSABLE(confused, expected)
+#endif
+
+CONFUSABLE(0x2010, 0x2d)
```

Comment:
It would help to describe the procedure you used to generate this table, so that it can be updated for a new version of Unicode when it comes out.

Please also mention in comments the Unicode version you used this time.